src/handle-data.ts

```diff
@@ -23,19 +35,27 @@ export function getColumnOrder({ qColumnOrder, qDimensionInfo, qMeasureInfo }) {
  * @param {Number} numDims
```

Comment:
you can remove the jsdocs now since the typing should add the same information. It is nice to keep a small description though

Reply:
sure